toggle-dummy.js

```diff
@@ -1,8 +1,27 @@
-  $(function(){
-    var button =
-      $('<button type="button" onclick="toggleDummy()"> Show dummies</button>');
-    $("body").prepend(button);
-  });
-  function toggleDummy(){
-    $(".dummy").toggleClass("is-active")
+$(function(){
+  addButton()
+  if (dummyOn()){
+    toggleDummy();
   }
+});
+
+function addButton() {
+  var button = $('<button type="button" onclick="toggleDummy();toggleStorage()"> Show dummies</button>');
+  $("body").prepend(button);
+}
+
+function toggleDummy(){
+  $(".dummy").toggleClass("is-active");
```

Comment:
What do you think about calling the `toggleStorage` here. I don't think you want these two function to be out of sync so you probably want to consider `toggleStorage` to be private to your "class".